### PR TITLE
EOS-27348: Utils Mini Provisioning Setup: Removed usage of cortxconf class from utils setup

### PR DESCRIPTION
**Pull request by suryakumar1024**
Signed-off-by: suryakumar.kumaravelan <suryakumar.kumaravelan@seagate.com>

# Problem Statement
- Since CortxConf class is no more in use, Need to update all the usage of this class all over the utils code.

# Design
-  Removed the cortxconf class usage in utils mini provisioning scripts

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: N
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: NA
- [x] Confirm Testing was performed with installed RPM [Y/N]:  Y

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [x] Is there a change in filename/package/module or signature [Y/N]: Y
- [x] If yes for above point, Is a notification sent to all other cortx components [Y/N]: Y
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -163,25 +145,25 @@ def config(config_path: str):
...
+            Conf.set('cortx_config', 'log_dir', log_dir)
+            Conf.save('cortx_config')
 
         # set cluster nodename:hostname mapping to cluster.conf
         Utils._copy_cluster_map(config_path)
         Utils._configure_rsyslog()
 
         # get shared storage from cluster.conf and set it to cortx.conf
-        shared_storage = CortxConf.get_storage_path('shared', none_allowed=True)
+        shared_storage = Conf.get(MappedConf._conf_idx, 'cortx>common>storage>shared')
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:158]: MappedConf must not be used as static class instance. You need to instantiate it to use.

File: py-utils/src/setup/utils.py [hunk 2]
```diff
@@ -163,25 +145,25 @@ def config(config_path: str):
         config_template_index = 'config'
         Conf.load(config_template_index, config_path)
         # Configure log_dir for utils
-        log_dir = CortxConf.get_storage_path('log')
+        log_dir = Conf.get(MappedConf._conf_idx, 'cortx>common>storage>log')
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:148]: MappedConf must not be used as static class instance. You need to instantiate it to use.

File: py-utils/src/setup/utils.py [hunk 3]
```diff
@@ -163,25 +145,25 @@ def config(config_path: str):
...
         if log_dir is not None:
-            CortxConf.set('log_dir', log_dir)
-            CortxConf.save()
+            Conf.set('cortx_config', 'log_dir', log_dir)
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:150]: It should be done this way 

cortx_conf  = MappedConf(cortx_config_url)
cortx_conf.set('log_dir', log_dir)

File: py-utils/src/setup/utils.py [hunk 4]
```diff
@@ -163,25 +145,25 @@ def config(config_path: str):
...
+        shared_storage = Conf.get(MappedConf._conf_idx, 'cortx>common>storage>shared')
         if shared_storage:
             Utils._set_to_conf_file('support>shared_path', shared_storage)
 
         # temporary fix for a common message bus log file
         # The issue happend when some user other than root:root is trying
         # to write logs in these log dir/files. This needs to be removed soon!
-        log_dir = CortxConf.get('log_dir', '/var/log')
-        utils_log_dir = CortxConf.get_log_path(base_dir=log_dir)
+        log_dir = Conf.get('cortx_config', 'log_dir', '/var/log')
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:165]: Use MappedConf. Refer to the above example.

File: py-utils/src/setup/utils.py [hunk 5]
```diff
@@ -270,7 +252,8 @@ def reset(config_path: str):
             raise SetupError(errors.ERR_OP_FAILED, "Internal error, can not \
                 reset Message Bus. %s", e)
         # Clear the logs
-        utils_log_path = CortxConf.get_log_path()
+        log_dir = Conf.get(MappedConf._conf_idx, 'cortx>common>storage>log')
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:255]: MappedConf must not be used as static class instance. You need to instantiate it to use.

File: py-utils/src/setup/utils.py [hunk 6]
```diff
@@ -306,7 +289,8 @@ def cleanup(pre_factory: bool, config_path: str):
 
         if pre_factory:
             # deleting all log files as part of pre-factory cleanup
-            utils_log_path = CortxConf.get_log_path()
+            log_dir = Conf.get(MappedConf._conf_idx, 'cortx>common>storage>log')
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils.py:292]: MappedConf must not be used as static class instance. You need to instantiate it to use.

File: py-utils/src/setup/utils_setup.py [hunk 7]
```diff
@@ -235,15 +236,13 @@ def process(self):
...
-    tmpl_file_index = 'tmpl_index'
+    from cortx.utils.conf_store import Conf, MappedConf
     argv = sys.argv
 
     # Get the log path
     tmpl_file = argv[3]
-    from cortx.utils.common import CortxConf
-    Conf.load(tmpl_file_index, tmpl_file, skip_reload=True)
-    local_storage_path = Conf.get(tmpl_file_index, 'cortx>common>storage>local')
+    clusterConfMapped = MappedConf(tmpl_file)
```
Comment (ujjwalpl) [inline py-utils/src/setup/utils_setup.py:244]: Use simple names. We can continue with existing name like this

cortx_conf = MappedConf(tmpl_file)

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/setup/utils.py (hunk 1 above)
Comment (suryakumar1024) [reply py-utils/src/setup/utils.py]: Addressed your comment in recent patch.

File: py-utils/src/setup/utils.py (hunk 2 above)
Comment (suryakumar1024) [reply py-utils/src/setup/utils.py]: Addressed your comment in recent patch.

File: py-utils/src/setup/utils.py (hunk 3 above)
Comment (suryakumar1024) [reply py-utils/src/setup/utils.py]: Addressed your comment in recent patch.

File: py-utils/src/setup/utils.py (hunk 4 above)
Comment (suryakumar1024) [reply py-utils/src/setup/utils.py]: Addressed your comment in recent patch.

File: py-utils/src/setup/utils.py (hunk 5 above)
Comment (suryakumar1024) [reply py-utils/src/setup/utils.py]: Addressed your comment in recent patch.

File: py-utils/src/setup/utils.py (hunk 6 above)
Comment (suryakumar1024) [reply py-utils/src/setup/utils.py]: Addressed your comment in recent patch.

File: py-utils/src/setup/utils_setup.py (hunk 7 above)
Comment (suryakumar1024) [reply py-utils/src/setup/utils_setup.py]: Addressed your comment in recent patch.

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by rohit-k-dwivedi — APPROVED** [review]

**Review by ujjwalpl — APPROVED** [review]

**Review by vastradparayya — APPROVED** [review]

**Pull request merged**